README.md

```diff
@@ -1,9 +1,9 @@
-# State channels
-State channels allow entities to communicate with each other with the goal of collectively computing some function `f`. This `f` can be as simple as "send 0.1 coins every minute" or it could represent a decentralised exchange. These functions are, in our case, represented by smart contracts and just like any legal contract, we need an arbiter in case one party tries to act maliciously. This arbiter is the blockchain. 
+# State Channels
+State Channels allow entities to communicate with each other with the goal of collectively computing some function `f`. This `f` can be as simple as "send 0.1 coins every minute" or it could represent a decentralised exchange. These functions are, in our case, represented by Smart Contracts and just like any legal contract, we need an arbiter in case one party tries to act maliciously. This arbiter is the blockchain. 
```

Comment:
Hmm, "send 0.1 coins every minute" isn't the best example, since contracts are idempotent, and have no sense of time. Of course, the State Channel participants to, so they could agree to call a function `f` every minute, and the agreement itself could be expressed in contract logic.

Reply:
I didn't change that part xD but let me think about rephrasing 🤔

Reply:
updated: https://github.com/aeternity/state-channel-demo/pull/251/commits/05a6d0eb197aadb204779fa2eb970eed1ba3605b